### PR TITLE
Fix version warnings

### PR DESCRIPTION
**Pull request by toxieainc**
At least the Intel GLSL compiler complains on each startup of Supermodel about the GLSL version missing in those shaders.

### CODE REVIEW

**Review by trzy — CHANGES_REQUESTED** [review]
Need to remove the CPU/ stuff from this as well as it has already been merged in.

File: Src/Graphics/New3D/R3DFrameBuffers.cpp [hunk 1]
```diff
@@ -269,10 +277,11 @@ void R3DFrameBuffers::AllocShaderTrans()
 		vec4 colTrans2 = texture2D( tex2, fsTexCoord);
 
 		if(colTrans1.a+colTrans2.a > 0.0) {
-			vec3 col1 = (colTrans1.rgb * colTrans1.a) / ( colTrans1.a + colTrans2.a);		// this is my best guess at the blending between the layers
-			vec3 col2 = (colTrans2.rgb * colTrans2.a) / ( colTrans1.a + colTrans2.a);
+			vec3 col1 = colTrans1.rgb * colTrans1.a;
```
[the remark this answers is not in the document]
Comment (toxieainc) [reply Src/Graphics/New3D/R3DFrameBuffers.cpp:280]: Just to save 3 or 4 instructions (unless the compiler is smart enough). The division is done some lines below (combined).

**Comment by toxieainc** [comment]
Also rebased all, so the Z80 changes are also gone here (sorry for that, should've created 3 separate branches in the 1st place :/).

**Comment by trzy** [comment]
Thanks, I want to give this a spin this weekend on some other systems and if all goes well will give final approval.

**Comment by toxieainc** [comment]
Could this be pushed? Its absolutely harmless and is pretty much equivalent to what is already there (minus triggering the missing version warnings ;))

**Comment by trzy** [comment]
Sure!

**Pull request merged**